edit authentication guides (#658)
vtexdocs/dev-portal-content: docs/guides/Getting-Started/authentication/user-authentication-and-login.md

* **Admin:** Administrative users' tokens allow for actions based on [License Manager roles](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc) attributed to them. This may include access to different Admin panel pages or the ability to edit different configurations, for example.

As described above, user tokens and associated permissions are tied to the user who logged in. However, they are not store-exclusive. This means, for example, that administrative users with access to different accounts can perform actions in all of those stores with the same token.

## Generating user tokens

Developers working with VTEX can generate authentication tokens without having to simulate a shopper’s login. This can be useful to run tests or even to generate tokens programmatically if your integration depends on an API that can not be authenticated with [application keys](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). There are two methods you can use:

- [Generating user tokens](#generating-user-tokens)

>     - [Generating an user token with the VTEX IO CLI](#generating-an-user-token-with-the-vtex-io-cli)

Reviewer: ```suggestion
    - [Generating a user token with the VTEX IO CLI](#generating-an-user-token-with-the-vtex-io-cli)
```

    - [Generating an authentication token with the VTEX ID API](#generating-an-authentication-token-with-the-vtex-id-api)
- [Single sign on integrations](#single-sign-on-integrations)

> #### Generating an user token with the VTEX IO CLI

Reviewer: ```suggestion
#### Generating a user token with the VTEX IO CLI
```

1. Install the [VTEX IO CLI](https://developers.vtex.com/docs/guides/vtex-io-documentation-vtex-io-cli-install).

> 2. Log in to VTEX by running this command on your terminal:

Reviewer: > Remember to replace the {accountName} value....

> 2. Log in to VTEX by running this command on your terminal:

Author: ```suggestion
2. Log in to VTEX by running the following command on your terminal.
   Remember to replace the `{accountName}` value with your VTEX account name.
```

* **Admin:** Administrative users' tokens allow for actions based on [License Manager roles](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc) attributed to them. This may include access to different Admin panel pages or the ability to edit different configurations, for example.

As described above, user tokens and associated permissions are tied to the user who logged in. However, they are not store-exclusive. This means, for example, that administrative users with access to different accounts can perform actions in all of those stores with the same token.

## Generating user tokens

Developers working with VTEX can generate authentication tokens without having to simulate a shopper’s login. This can be useful to run tests or even to generate tokens programmatically if your integration depends on an API that can not be authenticated with [application keys](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys). There are two methods you can use:

* [Generating user tokens](#generating-user-tokens)

>   * [Generating a user token with the VTEX IO CLI](#generating-an-user-token-with-the-vtex-io-cli)

Author: ```suggestion
  * [Generating a user token with the VTEX IO CLI](#generating-a-user-token-with-the-vtex-io-cli)
```

  * [Generating an authentication token with the VTEX ID API](#generating-an-authentication-token-with-the-vtex-id-api)
* [Single sign on integrations](#single-sign-on-integrations)

> #### Generating a user token with the VTEX IO CLI

Author: ```suggestion
### Generating a user token with the VTEX IO CLI
```

1. Install the [VTEX IO CLI](https://developers.vtex.com/docs/guides/vtex-io-documentation-vtex-io-cli-install).
2. Log in to VTEX by running the following command on your terminal.
   Remember to replace the `{accountName}` value with your VTEX account name.
   
   ```bash
   vtex login {accountName}
   ```

3. Once you are logged in to your VTEX account, run this command to generate a user token:

   ```bash
   vtex local token
   ```

   With this, the CLI will generate a valid user token associated with your profile, print it to your terminal and also copy it to your clipboard.

> #### Generating an authentication token with the VTEX ID API

Author: ```suggestion
### Generating an authentication token with the VTEX ID API
```
